.travis.yml

```diff
@@ -0,0 +1,40 @@
+sudo: required
+dist: trusty
+language: node_js
+
+# test with node 4.*
+node_js:
+- "4"
```

Comment:
There was some issues with installation under node5
I'll retest today and will make related changes